### PR TITLE
URL: Adding a README for the URL Package

### PR DESCRIPTION
**Pull request by youknowriad**
Everything is in the title

### CODE REVIEW

File: packages/url/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# WordPress Url Package
+
+A collection of utilities to manipulate URLs
+
+## Installation
+
+Install the module
+
+```bash
+npm install @wordpress/url --save
```
Comment (youknowriad) [inline packages/url/README.md]: I guess the name here is not defined yet, is it?

File: packages/url/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# WordPress Url Package
```
Comment (ntwb) [inline packages/url/README.md]: Capitalized `URL`

File: packages/url/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+npm install @wordpress/url --save
+```
+
+## Usage
+
+```JS
+import { addQueryArgs } from '@wordpress/url';
+
+// Appends arguments to the query string of a given url
+const newUrl = addQueryArgs( 'https://google.com', { q: 'test' } ); // https://www.google.dz/?q=test
```
Comment (ntwb) [inline packages/url/README.md]: The line comment here should also use `google.com` as per the example code.

Comment (youknowriad) [inline packages/url/README.md]: 🙃

File: packages/url/README.md (hunk 1 above)
Comment (aduth) [inline packages/url/README.md]: Not yet, but this is what it'd be published as.

File: packages/url/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+npm install @wordpress/url --save
+```
+
+## Usage
+
+```JS
+import { addQueryArgs } from '@wordpress/url';
+
+// Appends arguments to the query string of a given url
+const newUrl = addQueryArgs( 'https://google.com', { q: 'test' } ); // https://www.google.com/?q=test
```
Comment (aduth) [inline packages/url/README.md]: No `www` per example

Comment (ntwb) [inline packages/url/README.md]: Ha, I missed that 😉

**Pull request merged**